api/src/main/java/org/eclipse/microprofile/openapi/annotations/media/Schema.java

```diff
@@ -40,10 +54,7 @@
 public @interface Schema {
```

Comment:
What are your thoughts on having a `String value() default ""` property that could be used to set an arbitrary schema (that must be valid JSON/YAML)? That would give support for alternate dialects with the annotation.

I'm not sure myself, but I wanted to mention it as a possibility.

Reply:
That's certainly an interesting option which I hadn't considered. It becomes a bit more usable since Java 15 added text blocks too.

I'll raise a new issue to think about it.

Reply:
Opened #610 to continue the discussion after this is merged.